features/step_definitions/logout_steps.rb

```diff
@@ -1 +1,17 @@
-#Add your codez here
+Given(/^the secret page$/) do
+  within("#session") do
```

Comment:
Do you not have to visit the dashboard page first? This might not pass if you run just this test.